datasource/index.js

```diff
@@ -99,15 +103,16 @@ module.exports = yeoman.generators.Base.extend({
     var prompts = [
       {
         name: 'connector',
-        message: 'Select the connector for ' + displayName + ':',
+        message: g.f('Select the connector for %s:', displayName),
```

Comment:
Yup, this is the case of `string concatenation` globalization bug and  the right solution.